2.ServiceTracker_App/oam-remote.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: oam
+---
+apiVersion: stacks.crossplane.io/v1alpha1
+kind: ClusterStackInstall
+metadata:
+  name: addon-oam-kubernetes-remote
+  namespace: oam
+spec:
+  package: "crossplane/addon-oam-kubernetes-remote:master"
```

Comment:
could we use local stack and make it as default so we won't need this yaml file.

Reply:
> mark as default

Do you mean assume that the user already has the controller running? It seems to make sense to have an `oam-remote.yaml` and `oam-local.yaml` file here and include instructions for both in the `README.md`.

Reply:
Yeah, I assume `oam-local` along with crossplane are all default installed together.

Reply:
Hmm I don't know why we wouldn't provide instruction here for both cases. There currently is not documentation anywhere else for installing the remote and local addons so I think it makes sense to provide here.

Reply:
@wonderflow I removed this in favor of just mentioning that either the local or remote addon must be enabled :+1: